server/models/answers.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+// answer schema
+const answerSchema = new Schema({
+    author: {type: Schema.Types.ObjectId, ref: 'User'},
+    info:  {type: String},
+    text: {type: String, required: true},
+    comments: [{type: Schema.Types.ObjectId, ref: 'User'}],
+    date: {type: Date, 'default': Date.now},
+    scores: {type: Number, 'default': 0},
+    edited: {type: Boolean, 'default': false}
+});
+
+// build Answer model
+const Answer = mongoose.model('Answer', answerSchema);
+module.exports = Answer;
```

Comment:
files should have a blank line at the end of the file

Reply:
Fixed